### PR TITLE
remove oc dependency on internal build types

### PR DESCRIPTION
**Pull request by sanchezl**

### CODE REVIEW

**Comment by deads2k** [comment]
/hold 

check https://github.com/openshift/origin/pull/23095

**Comment by sanchezl** [comment]
/hold cancel

https://github.com/openshift/origin/pull/23095 merged

File: pkg/oc/cli/startbuild/startbuild_test.go [hunk 1]
```diff
@@ -27,7 +27,6 @@ import (
 
 	buildv1 "github.com/openshift/api/build/v1"
 	buildclientmanual "github.com/openshift/oc/pkg/helpers/build/client/v1"
-	buildapi "github.com/openshift/origin/pkg/build/apis/build"
 
 	_ "github.com/openshift/origin/pkg/build/apis/build/install"
```
Comment (soltysh) [inline pkg/oc/cli/startbuild/startbuild_test.go]: This is not addressed, and that's a major problem when you remove it that I'm currently handling.

**Pull request closed without merging**